src/chado_object/feature/chado_feature.py

```diff
@@ -409,11 +409,13 @@ def load_synonym(self, key: str, unattrib: bool = True, cvterm_name: str = None,
 
         for item in items:
             synonym_sgml = None
+            name = item[FIELD_VALUE]
             if 'subscript' in self.process_data[key] and not self.process_data[key]['subscript']:
```

Comment:
```suggestion
            if 'subscript' in self.process_data[key].keys() and self.process_data[key]['subscript'] is False:
```
I think something like the above suggestion is more explicit, and it may be helpful to someone like me who is unfamiliar with the code. But maybe I just need to get more familiar with your style, and I'm not suggesting it's a good idea to try and change similar lines throughout the code.

Reply:
I have created a new method subscript_convert which has the format as requested. This is reused in various places so hopefully will be less code repetition and easier to see what is going on.